cmd/elasticindexer/main.go

```diff
@@ -131,5 +147,15 @@ func initializeLogger(ctx *cli.Context, cfg *config.Config) (file.FileLoggingHan
 		return nil, err
 	}
 
+	err = logger.RemoveLogObserver(os.Stdout)
```

Comment:
might extract the new code in a function `removeANSICollorsForLogger`

Reply:
done.